src/utils/nextstrainAdapter.ts

```diff
@@ -72,6 +72,33 @@ export const initializeTree = (node: NSNode, parent: Node | "root") => {
   return newNode;
 };
 
+export const validateNextstrainJson = (nextstrainJSON: NSJSON) => {
+  // top level must have at least `tree` and `meta` as objects
+  if (
+    !(
+      Object.keys(nextstrainJSON).includes("tree") &&
+      Object.keys(nextstrainJSON).includes("meta")
+    )
+  ) {
+    throw new Error();
```

Comment:
It's usually best practice to put a message in your Error, simply to help debugging if the Error ever escapes and starts breaking stuff. You can throw anonymous errors like this, but you'd probably be better off putting in a concise, dev-facing message.
`throw new Error("tree JSON missing expected top level key")`, etc

Reply:
Fair. Done!